src/librustdoc/html/static/rustdoc.css

```diff
@@ -542,11 +542,12 @@ h4 > code, h3 > code, .invisible > code {
 }
 
 .content .stability::before {
-	content: '˪';
-	font-size: 30px;
+	content: '⤶';
+	transform:rotate(90deg);
```

Comment:
Also I'm not a fan of this. I'm pretty sure the expected character exists as is.

Reply:
I failed to find it. Will check that again.

Reply:
changed to another arrow: ⬑

![rotate-vs-nonrotate](https://user-images.githubusercontent.com/346530/72031158-808f2680-32c6-11ea-9419-97db43036e55.png)